src/main/webapp/ItemPage.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<meta charset="UTF-8" />
```

Comment:
Nit: The Google style guideline we are following mentions that the default indentation for HTML pages is 2 spaces instead of 4.